### DISCUSSION TITLE
动态过滤器能不能添加“留下包含某标签的动态”的功能？

### DISCUSSION THREAD

**Discussion by hydra13142** [the1812/Bilibili-Evolved#3390] [功能建议 / Ideas]
你看，现在的功能室过滤掉包含某标签的动态，只留下不包含该标签的动态；但也应该有留下包含某标签的动态这种功能更好吧？

比如说，我想从动态里搜索型月英灵的图片，但是呢我特烦金闪闪，所以我想搜索“型月，但是没有金闪闪”的图片，这个需求目前是做不到的。

**Comment by the1812** [comment]
这种就比较复杂了, 得接脚本提供的 `FeedsContentFilter` 自己写个插件
